portals/ghrc/config.json

```diff
@@ -1,20 +1,21 @@
 {
-    "hasStyles": true,
-    "hasScrpits": true,
-    "logo":{
-        "id": "ghrc-logo",
-        "link": "https://ghrc.nsstc.nasa.gov/home",
-        "title": "GHRC Home"
-    },
-    "org": "GHRC",
-    "pageTitle": "GHRC Portal",
-    "parentConfig": "edsc",
-    "query": {
-        "dataCenter": "NASA/MSFC/GHRC",
-        "hasGranulesorCwic": null
-    },
-    "ui": {
-        "showOnlyGranulesCheckbox": false,
-        "showNonEosdisCheckbox": false
-    }
-}
\ No newline at end of file
+  "hasStyles": true,
+  "hasScripts": true,
+  "logo": {
+    "id": "ghrc-logo",
+    "link": "https://ghrc.nsstc.nasa.gov/home",
+    "title": "GHRC Home"
+  },
+  "org": "GHRC",
+  "pageTitle": "GHRC Portal",
+  "parentConfig": "edsc",
+  "query": {
+    "dataCenter": "NASA/MSFC/GHRC",
+    "hasGranulesOrCwic": null
+  },
+  "ui": {
+    "showOnlyGranulesCheckbox": false,
+    "showNonEosdisCheckbox": false
+  }
+  }
```

Comment:
It looks like you changed all of your tabs/spaces for the entire PR which.. is fine but the last closing brace should not be indented here.

Reply:
done! thank you!